trac-env/htdocs/css/trachacks.scss

```diff
@@ -274,15 +274,17 @@ div[role="main"]{
       }
     }
 
-    .inlinebuttons input[type="submit"] {
+    .inlinebuttons input[type="submit"],
+    button.trac-revert {
       background: #ddd;
       color: #444;
       font-size: 1.1rem;
-    }
+      height: auto;
```

Comment:
I found it was necessary because the generic button style has `height: 46px` (from https://github.com/django/djangoproject.com/blob/master/djangoproject/scss/_style.scss#L2620).